src/Nethermind/Nethermind.Facade/Simulate/SimulateBridgeHelper.cs

```diff
@@ -253,10 +255,20 @@ private Transaction CreateTransaction(
             requestsHash: parent.RequestsHash)
         {
             MixHash = parent.MixHash,
-            IsPostMerge = parent.Difficulty == 0,
             RequestsHash = parent.RequestsHash,
         };
 
+        if ((ForkActivation)result.Number >= specProvider.MergeBlockNumber)
+        {
+            result.Difficulty = UInt256.Zero;
+            result.IsPostMerge = true;
+        }
+        else
+        {
+            result.Difficulty = parent.Difficulty;
+            result.IsPostMerge = false;
+        }
```

Comment:
In theory this is not the best way to check during the transition, we have `IPoSSwitcher` for that. But we could probably ignore that/

Reply:
I tried, but PosSwitcher is not expecting this type of usage

Reply:
Yeah probably not worth the effort, this is fine.